scrapers/base.py

```diff
@@ -46,3 +47,29 @@ def run(self, since=None):
         :raises NotImplementedError: if the function is not overridden
         """
         raise NotImplementedError()
+
+    def is_container_build(self, build_info):
+        """
+        Check whether a Koji build is a container build.
+
+        :param KojiBuild build_info: Build info from Teiid
```

Comment:
Optional: "Build" => "build" since parameter descriptions start out as lowercase throughout the rest of the code-base